src/services/members/search.js

```diff
@@ -10,15 +10,18 @@ const api = initializeApi({
 });
 
 export const searchMembers = async (searchTerm, searchParams) => {
-  const { start = 0, end = 50 } = searchParams;
+  const { start = 0, end = 50, roles = [], interests = [] } = searchParams;
   let response;
   try {
     response = await api({
       method: 'GET',
       url: urljoin(
         reactApiSearchMembersApi,
         'searchmembers',
-        `?queryString=${searchTerm}&start=${start}&end=${end}`,
+        `?queryString=${searchTerm}&start=${start}&end=${end}${roles
+          .map(role => `&role=${role}`).join('')}${interests
```

Comment:
I'd extract map callback in a method : both callback are essentially doing the same thing